openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -45,6 +91,19 @@ def test_get_account_different_user(self):
         self.different_client.login(username=self.different_user.username, password=TEST_PASSWORD)
         self.send_get(self.different_client, expected_status=404)
 
+    def test_get_account_anonymous_user(self):
```

Comment:
I accidentally moved them into the superclass, then moved them back, and obviously screwed things up in the process. Good catch!